### PR TITLE
For #1568: user resume page

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1568:
- Implemented resume passage to resumes page;
- Opened issue in `zerocracy/datum` for creating resumes page (https://github.com/zerocracy/datum/issues/392)
- Added puzzle to check if join page is being displayed correctly if user not joined yet

### CODE REVIEW

File: src/main/java/com/zerocracy/tk/TkJoin.java [hunk 1]
```diff
@@ -54,6 +51,30 @@ public TkJoin(final Farm frm) {
 
     @Override
     public Response act(final RqRegex req) throws IOException {
-        return new RsPage(this.farm, "/xsl/join.xsl", req);
+        final String author = new RqUser(this.farm, req, false).value();
+        final People people = new People(this.farm).bootstrap();
+        people.touch(author);
+        //@checkstyle FinalLocalVariableCheck (1 line)
+        RsPage result;
+        if (people.exists(author)) {
```
Comment (g4s8) [inline src/main/java/com/zerocracy/tk/TkJoin.java]: @paulodamaso we also should check that resume exist in `resumes.xml`

File: src/main/java/com/zerocracy/tk/TkJoin.java [hunk 2]
```diff
@@ -54,6 +51,30 @@ public TkJoin(final Farm frm) {
...
+                this.farm,
+                "/xsl/resume.xsl",
+                req,
+                () -> new XeAppend(
+                    "resume",
+                    new XeAppend("login", resume.login()),
+                    new XeAppend("text", resume.text()),
+                    new XeAppend("personality", resume.personality()),
+                    new XeAppend("soid", Long.toString(resume.soid())),
+                    new XeAppend("telegram", resume.telegram())
```
Comment (g4s8) [inline src/main/java/com/zerocracy/tk/TkJoin.java]: @paulodamaso let's show `examiner` of user as well

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) not enough funds available in the project, can't set budget of job [#1799](https://github.com/zerocracy/farm/issues/1799), see [§21](http://www.zerocracy.com/policy.html#21); @g4s8[/z](https://www.0crat.com/u/g4s8) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by paulodamaso** [comment]
@g4s8 Changes made, please take a look. For checking resume existence I had to make more work than the 30 min timeframe (I've already had worked in this task for making the PR, and made code implementation + tests in review corrections), so I'd be glad if you could boost this one.

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@paulodamaso thanks

**Comment by g4s8** [comment]
@0crat extra 20min @paulodamaso

**Comment by 0crat** [comment]
> @0crat extra 20min @paulodamaso  ([here](https://github.com/zerocracy/farm/issues/1799#issuecomment-442457772))

@g4s8 The project is under-funded, you can't do it now, see [§49](http://www.zerocracy.com/policy.html#49)

**Comment by paulodamaso** [comment]
@g4s8 Please take a look

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job [#1799](https://github.com/zerocracy/farm/issues/1799) is not in the agenda of @g4s8[/z](https://www.0crat.com/u/g4s8), can't retrieve data and time of add

**Comment by 0crat** [comment]
Job was finished in 26 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#1799](https://github.com/zerocracy/farm/issues/1799) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**